MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import WebKit
+
+protocol MiniAppCallbackProtocol: AnyObject {
+    func didRecieveScriptMessageResponse(messageId: String, response: String)
+    func didRecieveScriptMessageError(messageId: String, errorMessage: String)
+}
+
+internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
+
+    weak var delegate: MiniAppCallbackProtocol?
+    var hostAppMessageDelegate: MiniAppMessageProtocol?
+
+    init(delegate: MiniAppCallbackProtocol, hostAppMessageDelegate: MiniAppMessageProtocol) {
+        self.delegate = delegate
+        self.hostAppMessageDelegate = hostAppMessageDelegate
+        super.init()
+    }
+
+    public func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
```

Comment:
Can this public API go in its own extension?

Reply:
No, we can't. While adding a script handler to the user controller, we need a class that conforms to WKScriptMessageHandler. That's why I created a separate class MiniAppScriptMessageHandler